app/keymaps/darwin.json

```diff
@@ -36,6 +36,7 @@
   "editor:copy": "command+c",
   "editor:paste": "command+v",
   "editor:selectAll": "command+a",
+  "editor:search": "command+f",
```

Comment:
Might also be nice to bind ` Escape ` to close the search box.

Reply:
Whoops, I've been away for quite some time this took off <3. I'll try and make the changes later tonight.

Reply:
Well.. Apparently I 'later tonight' to me is almost a month later, but have no fear the changes have been implemented in b978a3. 

@kevinschaich @chabou @ppot rereview?